extensions/csharp/package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "vscode",
   "license": "MIT",
   "engines": {
-    "vscode": "0.10.x"
+    "vscode": "^0.10.0"
```

Comment:
We should avoid making updates to VS Code stuff unless necessary - please revert the changes to extensions coming from them